java/core/src/test/java/sleeper/core/partition/PartitionFactory.java

```diff
@@ -101,11 +101,11 @@ private Partition.Builder parent(List<Partition.Builder> children, String id, Re
     }
 
     private Region parentRegion(Region left, Region right) {
-        return Region.from(schema.getRowKeyFields().stream()
-                .map(field -> rangeFactory.createRange(field,
-                        left.getRange(field.getName()).getMin(),
-                        right.getRange(field.getName()).getMax()))
-                .collect(Collectors.toList()));
+        return new Region(schema.getRowKeyFields().stream()
+        .map(field -> rangeFactory.createRange(field,
+                left.getRange(field.getName()).getMin(),
+                right.getRange(field.getName()).getMax()))
+        .collect(Collectors.toList()));
```

Comment:
The indentation here looks a bit strange, i think lines 104-108 need to be indented 1 more. I guess checkstyle isnt picking it up since this is a test class?

Reply:
Fixed, thanks